src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -13,146 +13,198 @@
 import base64
 import os.path
 import re
+import sys
 import warnings
+from dataclasses import dataclass
 from typing import Union
-import sys
 
 import jsonc
```

Comment:
The `jsonc` module doesn't seem to handle comments well. If you comment out a line in `zowe.config.json` by prefixing it with a double slash (`//`), then `jsonc` fails to load it.

I know we had a discussion about `jsonc` packages before and there didn't seem to be many options that are actively maintained. I quickly tested the `commentjson` package and although it was last updated in 2020, it seems to handle JSON files that contain comments correctly.

Could you replace the `jsonc` import here and the reference to it in "requirements.txt" with the `commentjson` package instead? I'm also open to other ideas if there are alternative packages you think would be a better fit 🙂